src/Tribe/Aggregator/Errors.php

```diff
@@ -37,6 +37,7 @@ private function __construct() {
 
 		tribe_register_error( 'core:aggregator:invalid-create-record-type', __( 'An invalid import type was used when trying to create this import record.', 'the-events-calendar' ) );
 		tribe_register_error( 'core:aggregator:invalid-csv-file', __( 'You must provide a valid CSV file to perform a CSV import.', 'the-events-calendar' ) );
+		tribe_register_error( 'core:aggregator:invalid-facebook-token', __( 'We received an Invalid Facebook Token from the Servive.', 'the-events-calendar' ) );
```

Comment:
I think these errors are alphabetized. Can you move this one to the correct location?